version/distro/distro.go

```diff
@@ -30,6 +33,11 @@ const (
 	Gokrazy   = Distro("gokrazy")
 	WDMyCloud = Distro("wdmycloud")
 	Unraid    = Distro("unraid")
+	Fedora    = Distro("fedora")
+	Alpine    = Distro("alpine")
+	Void      = Distro("void")
+	Gentoo    = Distro("gentoo")
+	Unknown   = Distro("")
 )
```

Comment:
Can you update (er, add) the docs on the Distro type while you're here, warning that these consts aren't an exhaustive set, and warning about adding ubuntu or others?

And add an explicit line like:

```
Debian    = Distro("debian") // includes ubuntu, etc
```